aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -2175,6 +2187,56 @@ std::tuple<Tensor, Tensor> _symeig_helper_cuda(const Tensor& self, bool eigenvec
   }
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ linalg_eigh ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+// This is a type dispatch function for 'apply_magma_eigh'
+void linalg_eigh_magma(Tensor& eigenvalues, Tensor& eigenvectors, Tensor& infos, bool upper, bool compute_eigenvectors) {
```

Comment:
This name is a little misleading because it may dispatch to CPU after all. Maybe an extra comment upfront would be clearer.

There's no weird situation where PyTorch has MAGMA but not LAPACK, right?

Reply:
Seems like we don't need it, I just wanted to sanity check.